framework-plugins/lisk-framework-chain-connector-plugin/src/chain_connector_plugin.ts

```diff
@@ -682,6 +680,16 @@ export class ChainConnectorPlugin extends BasePlugin<ChainConnectorPluginConfig>
 					validatorsData.certificateThreshold >= BigInt(this._lastCertificate.height),
 			),
 		);
+		// Delete CCUs
+		// When given -1 then there is no limit
+		if (this._ccuSaveLimit !== -1) {
```

Comment:
Just curious, if `minimum` of `ccuSaveLimit` is set to 0, is it possible to be -1?

Reply:
I think it is safe to use `0` as no-limit? Since there should be at least 1 ccu to be saved?

Reply:
good point, minimum should be -1

Reply:
if someone doesn't want to save any CCU then there will be no option if we use `0` as no limit

Reply:
I think this option is similar to how many events we store, so somewhere it is consistent with that